CONTRIBUTING.md

```diff
@@ -30,8 +30,7 @@ See [our issue template](https://github.com/sinonjs/sinon/blob/master/.github/)
 
 Pick [an issue](http://github.com/cjohansen/Sinon.JS/issues) to fix, or pitch
 new features. To avoid wasting your time, please ask for feedback on feature
-suggestions either with [an issue](http://github.com/cjohansen/Sinon.JS/issues/new)
-or on [the mailing list](http://groups.google.com/group/sinonjs).
+suggestions with [an issue](http://github.com/cjohansen/Sinon.JS/issues/new).
```

Comment:
might as well update the repo, as well? => https://github.com/sinonjs/sinon/issues/new

Reply:
@fatso83 I've changed all the issue links in CONTRIBUTING.md